plugins/dependencies/src/test/resources/dependencies-versions-key-validated.txt

```diff
@@ -603,21 +603,23 @@ io.kotest..kotest-extensions-koin=version.kotest
 io.kotest..kotest-extensions-mockserver=version.kotest
 io.kotest..kotest-extensions-spring=version.kotest
 io.kotest..kotest-extensions-testcontainers=version.kotest
+io.kotest..kotest-framework-api=version.kotest
+io.kotest..kotest-framework-datatest=version.kotest
 io.kotest..kotest-plugins-pitest=version.kotest
 io.kotest..kotest-property-arrow=version.kotest
 io.kotest..kotest-property=version.kotest
 io.kotest..kotest-runner-junit4=version.kotest
 io.kotest..kotest-runner-junit5=version.kotest
-io.kotest.extensions..kotest-extensions-allure=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-embedded-kafka=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-gherkin=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-koin=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-mockserver=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-pitest=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-robolectric=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-spring=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-testcontainers=version.kotest-extensions
-io.kotest.extensions..kotest-extensions-wiremock=version.kotest-extensions
+io.kotest.extensions..kotest-extensions-allure=version.NO-RULE
+io.kotest.extensions..kotest-extensions-embedded-kafka=version.NO-RULE
+io.kotest.extensions..kotest-extensions-gherkin=version.NO-RULE
+io.kotest.extensions..kotest-extensions-koin=version.NO-RULE
+io.kotest.extensions..kotest-extensions-mockserver=version.NO-RULE
+io.kotest.extensions..kotest-extensions-pitest=version.NO-RULE
+io.kotest.extensions..kotest-extensions-robolectric=version.NO-RULE
+io.kotest.extensions..kotest-extensions-spring=version.NO-RULE
+io.kotest.extensions..kotest-extensions-testcontainers=version.NO-RULE
+io.kotest.extensions..kotest-extensions-wiremock=version.NO-RULE
```

Comment:
As you can see, there's no rule remaining here, which means it would default back to the longer `version.maven-group..name` syntax, defeating the purpose of your intention in this PR.

Reply:
That's the point of the PR. `io.kotest.extensions:kotest-extensions-wiremock` is not tied to any other extension's lifecycle, nor the main kotest lifecycle. So it should be treated independently

Reply:
The other alternative would be for each extension to resolve to a nicer name, like `version.kotest.extension.allure`, `version.kotest.extension.spring`, etc.

Reply:
I think I understand. Every extension is supposed to have a separate, independent version, is that right?

In that case, yes to adding a rule that will make the version key a bit nicer.